bundle/upstream/manifests/quayregistries.crd.yaml

```diff
@@ -53,6 +53,28 @@ spec:
           status:
             description: QuayRegistryStatus defines the observed state of QuayRegistry.
             properties:
+              componentConditions:
+                description: Component Conditions shows the conditions in a per component basis
+                type: object
+                additionalProperties:
```

Comment:
Yes, we could. Here the component name would be the index and as the list of components is a slice I opted to leave this as is. If we go that way then it would make sense to also change the way we set components to managed or not and use a schema there as well. WDYT?

Reply:
Regardless of what we are going to do with `Spec.Components` I adjusted the status to have a schema. PTAL.